tests/api_test.py

```diff
@@ -114,6 +124,55 @@ def test_create_newdle_duplicate_timeslot(flask_client, dummy_uid):
     }
 
 
+@pytest.mark.usefixtures('db_session')
+def test_create_newdle_participant_email_sending(flask_client, dummy_uid):
+    django_mail.outbox = []
```

Comment:
this could be in a `autouse=True` fixture ;)

Reply:
Even better, a named fixture you can use wherever you wish and which returns you the actual inbox list:
```python
@pytest.fixture
def mail_queue():
    from newdle.vendor import django_mail
    django_mail.outbox = []
    return django_mail.outbox
```